Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -148,6 +166,10 @@ int main(void)
     /* USER CODE BEGIN RTOS_THREADS */
     //TODO: Create threads and thread attributes
     /* add threads, ... */
+    const osThreadAttr_t blueTask_attributes = {.name = "blueTask", .priority = (osPriority_t) osPriorityNormal, .stack_size = 128};
+    const osThreadAttr_t greenTask_attributes = {.name = "greenTask", .priority = (osPriority_t) osPriorityNormal, .stack_size = 128};
+    blueThread = osThreadNew((osThreadFunc_t)blueLedToggleTask, &canMutexAttr, &blueTask_attributes);
```

Comment:
You are passing in the value for the mutex attributes instead instead of the handle